src/apim_payment_manager.tf

```diff
@@ -0,0 +1,541 @@
+resource "azurerm_api_management_api_version_set" "buyerbanks_api" {
+
+  name                = format("%s-buyerbanks-api", var.env_short)
+  resource_group_name = azurerm_resource_group.rg_api.name
+  api_management_name = module.apim.name
+  display_name        = local.apim_buyerbanks_api.display_name
+  versioning_scheme   = "Segment"
+}
+
+module "apim_buyerbanks_api_v1" {
+
+  source = "git::https://github.com/pagopa/azurerm.git//api_management_api?ref=v1.0.16"
+
+  name                  = format("%s-buyerbanks-api", var.env_short)
+  api_management_name   = module.apim.name
+  resource_group_name   = azurerm_resource_group.rg_api.name
+  product_ids           = [module.apim_payment_manager_product.product_id]
+  subscription_required = local.apim_buyerbanks_api.subscription_required
+  version_set_id        = azurerm_api_management_api_version_set.buyerbanks_api.id
+  api_version           = "v1"
+  service_url           = local.apim_buyerbanks_api.service_url
+
+  description  = local.apim_buyerbanks_api.description
+  display_name = local.apim_buyerbanks_api.display_name
+  path         = local.apim_buyerbanks_api.path
+  protocols    = ["https"]
+
+  content_format = "swagger-json"
+  content_value = templatefile("./api/payment_manager_api/buyerbanks/_swagger.json.tpl", {
+    host = azurerm_api_management_custom_domain.api_custom_domain.proxy[0].host_name
+  })
+
+  xml_content = file("./api/payment_manager_api/buyerbanks/_base_policy.xml.tpl")
+}
+
+#####################################
+## API restapi                     ##
+#####################################
+locals {
+  apim_pm_restapi_api = {
+    # params for all api versions
+    display_name          = "Payment Manager restapi API"
+    description           = "API to support payment trasactions for Checkout and Wisp"
+    path                  = "payment-manager/restapi"
+    subscription_required = false
+    service_url           = null
+  }
+}
+
+data "azurerm_key_vault_secret" "pm_restapi_ip" {
+  name         = "pm-restapi-ip"
+  key_vault_id = module.key_vault.id
+}
+
+resource "azurerm_api_management_api_version_set" "pm_restapi_api" {
+
+  name                = format("%s-pm-restapi-api", local.project)
+  resource_group_name = azurerm_resource_group.rg_api.name
+  api_management_name = module.apim.name
+  display_name        = local.apim_pm_restapi_api.display_name
+  versioning_scheme   = "Segment"
+}
+
+module "apim_pm_restapi_api_v4" {
+
+  source = "git::https://github.com/pagopa/azurerm.git//api_management_api?ref=v1.0.16"
+
+  name                  = format("%s-pm-restapi-api", local.project)
+  api_management_name   = module.apim.name
+  resource_group_name   = azurerm_resource_group.rg_api.name
+  product_ids           = [module.apim_payment_manager_product.product_id]
+  subscription_required = local.apim_pm_restapi_api.subscription_required
+  version_set_id        = azurerm_api_management_api_version_set.pm_restapi_api.id
+  api_version           = "v4"
```

Comment:
why directly `v4` and not `v1` ?

Reply:
in order to descrive the API version currently active for `restapi`, so only v4